### PR TITLE
Update ToolBar.svelte

### PR DESCRIPTION
**Pull request by bhiwagade-rahul**
Remove Solve and Step from Sketch Actions

### CODE REVIEW

File: applications/web/src/components/ToolBar.svelte [hunk 1]
```diff
@@ -7,7 +7,6 @@
 
   let solving = false
   // todo ask Matt why is this a no-op?
```
Comment (av8ta) [inline applications/web/src/components/ToolBar.svelte:9]: Some more things you could remove here

Comment (av8ta) [reply applications/web/src/components/ToolBar.svelte:9]: And further down in the file too.

**Review by av8ta — CHANGES_REQUESTED** [review]

File: applications/web/src/components/ToolBar.svelte (hunk 1 above)
Comment (bhiwagade-rahul) [reply applications/web/src/components/ToolBar.svelte:9]: @av8ta  plz review I have made few more changes.

**Comment by av8ta** [comment]
Sorry @bhiwagade-rahul I'm just going to go ahead and close this because it's more work teaching you what to do than it would be to just make the change. Hope you understand, we're a small team so we don't have time for this.

**Comment by bhiwagade-rahul** [comment]
@av8ta Thanks for the feedback.

**Pull request closed without merging**